frontend/src/components/navigator/BallIcon.jsx

```diff
@@ -0,0 +1,20 @@
+import React from "react";
+
```

Comment:
youre gunna have to walk me through this stuff, or send me somewhere I can read up on SVGs